app/controllers/groups_controller.rb

```diff
@@ -2,7 +2,7 @@ class GroupsController < PublicFacingController
   def show
     @organisation = Organisation.friendly.find(params[:organisation_id])
     @group = @organisation.groups.find_by!(slug: params[:id])
-    @group_members = @group.members.map do |p|
+    @group_members = @group.members.order('group_memberships.id').map do |p|
```

Comment:
We could instead add this into the join clause to turn it into the "Default ordering" explicitly, rather than just in this case.

For example, there's now no guarantee that these will be displayed in this order in the admin system, but there *is* an order guarantee in the frontend. To avoid this, if we put the order clause on the `has_many :group_memberships` clause against the model, we don't have to add the order in the second place.